manifest.json

```diff
@@ -669,6 +669,28 @@
               "variant": "v8"
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/windows-nano/1803/helix/amd64",
+              "os": "windows",
```

Comment:
One piece of special metadata that is needed for Window images is `"osVersion": "nanoserver-1803",`